ofrak_patch_maker/ofrak_patch_maker_test/test_toolchain_c.py

```diff
@@ -121,10 +130,16 @@ def test_hello_world(toolchain: ToolchainVersion, proc: ProgramAttributes, exten
     source_path = os.path.join(source_dir, "hello_world.c")
     build_dir = tempfile.mkdtemp()
 
+    if toolchain in [ToolchainVersion.GNU_AVR_5]:
+        relocatable = False
```

Comment:
If the AVR compiler cannot support relocatable code please leave a comment to that effect here and raise in the AVR toolchain `__init__` if the relocatable flag is set to true.

Reply:
Added a comment here